projects/ngx-charts-on-fhir/src/lib/fhir-data/fhir-data.service.ts

```diff
@@ -94,6 +99,82 @@ export class FhirDataService {
       return teardownLogic;
     });
   }
+
+  addPatientData(resource: fhirclient.FHIR.Resource) {
+    return this.client?.create(resource);
+  }
+
+  createResourceData(reportBPValue: BloodPressure): fhirclient.FHIR.Resource {
+    return {
+      "resourceType": "Observation",
```

Comment:
Can we make this method name more specific? How about `createBloodPressureResource()`